tests/framework/cli/pipeline/test_pipeline_pull.py

```diff
@@ -631,6 +702,7 @@ def test_pipeline_pull_all(  # pylint: disable=too-many-locals
         self, fake_repo_path, fake_project_cli, fake_metadata, mocker
     ):
         # pylint: disable=import-outside-toplevel
+        # pylint: disable=line-too-long
```

Comment:
AFAIK `black` should reformat the file to break up the line, so you shouldn't have to disable this.